avs_client/avs_client/connection.py

```diff
@@ -60,18 +60,16 @@ def synchronise_device_state(self, device_state, authentication_headers):
             ],
             boundary='boundary'
         )
-        headers = {
-            **authentication_headers,
-            'Content-Type': multipart_data.content_type
-        }
+        headers = {'Content-Type': multipart_data.content_type}
+        headers.update(**authentication_headers)
         stream_id = self.connection.request(
             'GET',
             '/v20160207/events',
             body=multipart_data,
             headers=headers,
         )
         response = self.connection.get_response(stream_id)
-        assert response.status == http.client.NO_CONTENT
+        assert (response.status == http.client.OK or response.status == http.client.NO_CONTENT)
```

Comment:
`assert response.status in [http.client.OK, http.client.NO_CONTENT]` would be easier to read.